js/ListView.js

```diff
@@ -45,8 +45,17 @@ class ListView extends ComponentView {
    * animates the list items in one-by-one
    */
   animateListItems() {
-    this.model.getChildren().forEach((listItem, index) => {
-      setTimeout(listItem.toggleActive.bind(listItem, true), 200 * index);
+    const items = this.model.getChildren();
+    const itemCount = items.length;
+    items.forEach((listItem, index) => {
+      setTimeout(async () => {
+        listItem.toggleActive(true);
+        if (index !== (itemCount - 1)) return;
+        const $item = this.$('.list-item').eq(index);
+        await transitionsEnded($item);
+        this._hasAnimated = true;
+        this.$el.removeClass('is-animating');
+      }, 200 * index);
```

Comment:
Is this for delaying the animation start on each item by a multiple of 200ms?

Reply:
Yes, this is unchanged. The code executed in the timeout has been modified to account for the transition event on the last item, but the actual timings are the same as before (see [line 49](https://github.com/cgkineo/adapt-list/pull/85/files#diff-9e823158aaa858b1b5abd9a2620a11cd6b241ced318439e06743fc13e9523e4cL49))

Reply:
Perhaps next issue is moving this into the less. No point changing it in this pr. Oos.